app/helpers/embed_player_helper.rb

```diff
@@ -93,6 +93,12 @@ def embed_player_category_options(podcast, selected)
     options_for_select(opts, selected)
   end
 
+  def enclosure_with_token(ep)
```

Comment:
Moved this from private->public, so I could test it.  (This helper had no tests, but this seemed worth testing).